README.md

```diff
@@ -60,24 +55,20 @@ The following peer dependencies are added to `package.json`:
 | @angular/material | ^13.0.0 |
 | @angular/cdk      | ^13.0.0 |
 
-## Running the demo projects
+## Running the Demo Projects
 
-This repository comes with two demo projects found within the `/demos` folder.
-The first is a [Storybook](https://storybook.js.org/) application that allows you to see the components in isolation and interact with their properties. The second is a Showcase project that shows a combination of components in the context of a realistic interface.
-
-You can build, link, and start the demo applications in a single step by calling either
+To start the documentation site that is included in this repository, run:
```

Comment:
Add a link and a screenshot would help

Reply:
Same for the showcase project instruction below

Reply:
Ok I've added links to both demo projects.